src/lib/PostgresMetaTypes.ts

```diff
@@ -11,23 +11,35 @@ export default class PostgresMetaTypes {
   }
 
   async list({
+    includeArrayTypes = false,
     includeSystemSchemas = false,
     limit,
     offset,
   }: {
+    includeArrayTypes?: boolean
     includeSystemSchemas?: boolean
     limit?: number
     offset?: number
   } = {}): Promise<PostgresMetaResult<PostgresType[]>> {
     let sql = typesSql
+    if (!includeArrayTypes) {
+      sql += ` and not exists (
+                 select
```

Comment:
```suggestion
                 select
                   1
```

missing select 1? https://github.com/supabase/postgres-meta/pull/423/files#diff-036b3b055e43dc27784cbae9d673e7512eb49cefa7b7d74568bc85aad297a59bL36

Reply:
It seems to be optional inside `exists`